examples/decorator/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "decorator"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "decorator"
+crate-type = ["cdylib"]
+
+[dependencies.pyo3]
+# If you copy this example, you should uncomment this...
+# version = "0.15.1"
+
+# ...and delete this path
+path = "../.."
+features = ["extension-module"]
+
+[workspace]
```

Comment:
Presumably these `[workspace]` keys are to trick cargo into treating these as new workspaces, now that we've removed these from the `Cargo.toml`?

Maybe users don't need to copy that either

```suggestion
features = ["extension-module"]
# If you copy this example, you should uncomment this...
# version = "0.15.1"

# ...and delete everything below here until the end of the file
path = "../.."

[workspace]
```

Reply:
Yes, it is to make cargo play nice.

If an user clones the entire pyo3 repository and starts working in the example crate, do they still need the `[workspace]` key?

I'm not entirely sure. Is there some git command to create a repository containing only the example crate? Other than us using git submodules, which I guess we could do...

I have to say I'm not a fan of "to use this example, change this and that and this". I would really like something that Just Works straight out of the box.

Reply:
That's fair. How about #2051 to solve the user issue?